.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: "Updatecli: Dependency Management"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '15 */2 * * *'
+  repository_dispatch:
+    types: [kubewarden-controller-release]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  updatecli:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
```

Comment:
I think we don't need node (and repair npm, install dependencies). Installing updatecli should be enough.

Reply:
I've removed them